test/basic.js

```diff
@@ -549,6 +577,8 @@ util.testSinglePlatform('ignore test: string in array', createIgnoreTest, ['igno
 util.testSinglePlatform('ignore test: string', createIgnoreTest, 'ignorethis', 'ignorethis.txt')
 util.testSinglePlatform('ignore test: RegExp', createIgnoreTest, /ignorethis/, 'ignorethis.txt')
 util.testSinglePlatform('ignore test: Function', createIgnoreTest, function (file) { return file.match(/ignorethis/) }, 'ignorethis.txt')
+util.testSinglePlatform('ignore test: Function', createIgnoreFromGlobalFileTest, 'ignorethis.txt')
```

Comment:
Keep in mind when you refactor your code to also add a test for the platform-specific ignores as well.